great-idea/css/index.css

```diff
@@ -63,4 +63,396 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
```

Comment:
The design files have the page in narrow container. To replicate that here, you can use a `container` div with a fixed width.